README.rst

```diff
@@ -103,39 +98,43 @@ Tests and coverage can be run using Pytest.
 
 .. note::
 
-    For the QIR compiler and loader to work the PyQIR (v0.9.0) is required. It can be
-    installed manually with ``pip install pyqir==0.9.0`` or as an optional dependency:
+    For the QIR compiler and loader to work the PyQIR (v0.9.0) is required. It
+    can be installed manually with ``pip install pyqir==0.9.0`` or as an
+    optional dependency:
 
     .. code-block:: bash
 
         pip install dwave-gate[qir]
 
-.. installation-end-marker
 
 License
--------
+=======
 
 Released under the Apache License 2.0. See LICENSE file.
 
+
 Contributing
-------------
+============
 
-Ocean's `contributing guide <https://docs.ocean.dwavesys.com/en/stable/contributing.html>`_
+Ocean's `contributing guide <https://docs.dwavequantum.com/en/latest/ocean/index_ocean_sdk.html>`_
 has guidelines for contributing to Ocean packages.
 
 Release Notes
-~~~~~~~~~~~~~
+-------------
 
-``dwave-gate`` uses `reno <https://docs.openstack.org/reno/>`_ to manage its release notes.
+``dwave-gate`` uses `reno <https://docs.openstack.org/reno/>`_ to manage its
+release notes.
 
-When making a contribution to ``dwave-gate`` that will affect users, create a new release note file
-by running
+When making a contribution to ``dwave-gate`` that will affect users, create a
+new release note file by running
 
 .. code-block:: bash
 
     reno new your-short-descriptor-here
 
-You can then edit the file created under ``releasenotes/notes/``. Remove any sections not relevant
-to your changes. Commit the file along with your changes.
+You can then edit the file created under ``releasenotes/notes/``. Remove any
+sections not relevant to your changes. Commit the file along with your changes.
 
-See reno's `user guide <https://docs.openstack.org/reno/latest/user/usage.html>`_ for details.
+See reno's
+`user guide <https://docs.openstack.org/reno/latest/user/usage.html>`_ for
```

Comment:
If we link to the latest reno docs, we should probably have the `requirements_dev.txt` install the latest as well, but strangely enough I cannot find any stable docs to point to (so it probably doesn't matter much).